### PR TITLE
[Review] New Page - Projects

### PR DESCRIPTION
**Pull request by darkdrgn2k**
Added a preliminary page of currently projects.

Please review/edit/update/whatever else you wish to do to update/correct etc

Pull It, Correct It, Commit it

### CODE REVIEW

**Comment by darkdrgn2k** [comment]
References Issue #137

File: projects.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+parent: Get Involved
+order: 1
+---
+
+Please ask in #welcome:tomesh.net if you would like to learn more or contribute to any one of these projects.
+
+## Nodes/Network
+
+### Raspberry/Orange Pi Install Script + Bi-Weekly Virtual Mesh
+Description: Installation script for Raspbery and Orange Pi devices from a stock firmware.
```
Comment (dcwalk) [inline projects.md]: Can we lint this? Not proper syntax in some places

File: projects.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+title: Projects
+parent: Get Involved
+order: 1
+---
+
+Please ask in #welcome:tomesh.net if you would like to learn more or contribute to any one of these projects.
+
+## Nodes/Network
+
+### Raspberry/Orange Pi Install Script + Bi-Weekly Virtual Mesh
```
Comment (dcwalk) [inline projects.md]: I don't know if we want recurring events listed as a project, not sure on that tho

File: projects.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Description: Installation script for Raspbery and Orange Pi devices from a stock firmware.   
+Status: See [Github Issues](https://github.com/tomeshnet/prototype-cjdns-pi/issues)  
+Contact: benhylau  
+[More Info](https://github.com/tomeshnet/prototype-cjdns-pi2)
+
+
+### Mesh Orange
```
Comment (dcwalk) [inline projects.md]: I think we want more discriptive names, like "Build low-cost mesh nodes using a mesh orange"

File: projects.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+Description: Ramdisk based os for keeping a lean and mean node on Orange Pis  
+Status: Work in progess https://github.com/tomeshnet/mesh-orange/issues/1  
+Contact: benhylau
```
Comment (dcwalk) [inline projects.md]: Could we link to user names in some way? (maybe github?) (indicate talking to people on Chat or where convos happen?)

File: projects.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+Status: Work in progess https://github.com/tomeshnet/mesh-orange/issues/1  
+Contact: benhylau  
+[More Info](https://github.com/tomeshnet/mesh-orange)  
+
+### Mesh ISP
+Status: Ethermesh did some demo of this at the conference and one of the mesh locals in the US is doing something similar  
+Contact: benhylau (until someone takes over)  
+[More Info](https://github.com/tomeshnet/mesh-isp)  
+
+### Outernet / RachelPi OS
```
Comment (dcwalk) [inline projects.md]: I think we need to rank these in terms of activity?/progress?/priority?

File: projects.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+Contact: garry  
+https://github.com/tomeshnet/meshbot  
+https://chat.tomesh.net/#/room/#bots:tomesh.net  
+
+### Tools
+Status: benhylau would really like some devops help, see Github Issues on https://github.com/tomeshnet/documents/issues  
+Contact: benhylau  
+https://chat.tomesh.net/#/room/#tools:tomesh.net  
+
+### Reading Networks
```
Comment (dcwalk) [inline projects.md]: This isn't active

File: projects.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+More Info: [RachelOffline](https://racheloffline.org)  
+
+## Resources
+
+### Website
+Status: See Github Issues  
+Contact: garry  
+[More Info](https://github.com/tomeshnet/tomesh.net)  
+
+#### Stripe
```
Comment (dcwalk) [inline projects.md]: Again, not sure if this is a "project" in its own right

File: projects.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+Contact: benhylau (until someone takes over)  
+[More Info](https://github.com/tomeshnet/mesh-isp)  
+
+### Outernet / RachelPi OS
+Status: No one is actively working on this but benhylau tried to set up Outernet with charlie from nycmesh, and spoke with someone from sudomesh about hosting wikipedia and khan with RACHEL  
+Contact: benhylau (until someone takes over)  
+More Info: [OuterNet](https://outernet.is)  
+More Info: [RachelOffline](https://racheloffline.org)  
+
+## Resources
```
Comment (dcwalk) [inline projects.md]: Unclear how this section is different than the "Get Involved" parent page?

**Review by dcwalk — COMMENTED** [review]
Left some initial thoughts, I think we might want to work on copy/text in a more collaborative format like a riseup/cryptpad

**Comment by dcwalk** [comment]
I've pasted into one here: https://cryptpad.fr/code/#/1/edit/iyLw65CNaPfRVpLy+vOrgA/p0QYKlNYLq-KjLw1o4rYIbt-/

Also would be good to sync this up with the Wekan

**Comment by darkdrgn2k** [comment]
Wekan: We need to either UPDATE Wekan, or push it into the background (maybe internal tool). Right now its very very outdated.

As for cyrpto pad, this list was originally taken from crypto pad in issue #137  Figured a PR would make it more "REAL" and maybe refresh peoples minds that we need to get this done :)

File: projects.md (hunk 5 above)
Comment (darkdrgn2k) [reply projects.md]: I area, maybe just list the ACTIVE projects here and leave the ideas some where else or lower on the page?

File: projects.md (hunk 8 above)
Comment (darkdrgn2k) [reply projects.md]: I think get involved needs some TLC as well and make it allot easier to follow. The detailed stuff (like how to use WEKA) may be worth having some where else. Dont want to scare people off when they first come here:)

**Comment by dcwalk** [comment]
@darkdrgn2k just wondering if this is something you are interested in putting more time into? I did a pass of updating the wekan to reflect project status last week I think...? But I'm still feeling like some drafting might be to happen pre-github PR

**Review by benhylau — CHANGES_REQUESTED** [review]
I suggest closing this PR since it is just a paste of an out of date and poorly formatted pad. If someone wants a Projects page, a PR with properly formatted file containing active projects should be submitted.

**Comment by dcwalk** [comment]
I second that @benhylau. Maybe we close by the end of the week unless someone commits to seeing this through?

**Comment by darkdrgn2k** [comment]
I cleaned up the page a bit but its still far from being "good to go".

> Unclear how this section is different than the "Get Involved" parent page?

I thought it be nice to outline some of the diverse projects we have going on as many people are under the initial impression that it is only a tech project. It may get more people interested looking at the diverse issues we are attacking.

However I'm also unsure if this is the best way of doing it. Thoughts? 

- [ ]  Keep it 
or
 - [ ] Scrap it

IF KEEP


>Again, not sure if this is a "project" in its own right 

>This isn't active

> I think we need to rank these in terms of activity?/progress?/priority?

Maybe we need to review which projects we want to highlight (we think are important/need help with)  here instead of recreating Wekan

But if that's the course we are taking i think Wekan needs to be a bit more prominent then in the get involved page, its lost right now inside the text.

Also whats the process of actually getting write access to the page? not sure few people asked in the chat and i didnt get an answer.

> I think we want more descriptive names, like "Build low-cost mesh nodes using a mesh orange"

I think that would have to come from the people on the project. Some of these projects I have not heard of others I may be under a different impression of what they are then what the organizers think they should be.

Again if you want to scrap it im down for that.

**Comment by dcwalk** [comment]
Okay, it looks like we have 3 "scrap for now" so I'm closing this PR. Maybe later in the spring we'd aim to revisit with website materials more broadly?

**Pull request closed without merging**